images/mediatypes.go

```diff
@@ -49,6 +49,9 @@ const (
 	MediaTypeContainerd1CheckpointRuntimeOptions = "application/vnd.containerd.container.checkpoint.runtime.options+proto"
 	// Legacy Docker schema1 manifest
 	MediaTypeDockerSchema1Manifest = "application/vnd.docker.distribution.manifest.v1+prettyjws"
+	// Encypted media types
+	MediaTypeImageLayerEncrypted     = ocispec.MediaTypeImageLayer + "+encrypted"
+	MediaTypeImageLayerGzipEncrypted = ocispec.MediaTypeImageLayerGzip + "+encrypted"
```

Comment:
FYI. There is a definition of the media type in https://github.com/containers/ocicrypt/blob/master/spec/spec.go. I'm guessing it may cause more dependencies for building certain builds though, so not sure what the implications there are if imported.

Reply:
I'd prefer not to add Go dependency on ocicrypt here

Reply:
Mark it as followup ~